packages/synapse-react-client/src/components/SynapseTable/SynapseTableRenderers.tsx

```diff
@@ -45,43 +45,45 @@ function RowSelectionCell(props: CellContext<Row, unknown>) {
   const [selectedRows, setSelectedRows] = useAtom(selectedRowsAtom)
 
   return (
-    <Checkbox
-      label=""
-      checked={isRowSelected(row.original)}
-      onChange={(checked: boolean) => {
-        const cloneSelectedRows = [...selectedRows]
-        if (checked) {
-          cloneSelectedRows.push(row.original)
-        } else {
-          const index = cloneSelectedRows.findIndex(selectedRow =>
-            isEqual(selectedRow, row.original),
-          )
-          if (index > -1) {
-            cloneSelectedRows.splice(index, 1)
+    <div>
```

Comment:
wrap in a div so the CSS that defines padding for all table elements adds to the `.checkbox` class padding instead of replacing it